examples/gbasf2_fei_training/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "gbasf2_install_directory": "/work/akhmet/Belle2Project/gbasf2",
```

Comment:
When you finalize the PR it would be better to leave all settings that are specific to you personal setup at the default values. E.g. `gbasf2_install_directory`, `result_dir`, etc. Though it should be clear anyway that this is just an example

Reply:
I would prefer to leave the settings explicit - changed to some dummy paths to be replaced, of cource - since I plan to refer to them in the documentation of this example.

I'll do that in one of the following commits

Reply:
I understand that, so it's to showcase the settings where have available. I admit settings discoverability is bad at the moment.

Reply:
Ok created a proper template-like example for settings